elasticapm/contrib/django/client.py

```diff
@@ -188,15 +188,6 @@ def _get_stack_info_for_trace(self, frames,
             exclude_paths_re=self.exclude_paths_re,
             locals_processor_func=locals_processor_func,
         ))
-        i = 0
-        while len(frames) > i:
-            if 'module' in frames[i] and not (
-                    frames[i]['module'].startswith('elasticapm.') or
-                    frames[i]['module'] == 'contextlib'
```

Comment:
what happens with `contextlib`, don't we want to filter it out anymore?

Reply:
I think that's a relic from the olden days. We don't use `contextlib` anywhere in the code base